.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,26 +2,17 @@
  Please fill this template unless your PR is very simple/straightforward.
  Do not forget to have a look at our Pull Request tutorial: http://www.azerothcore.org/wiki/Contribute#how-to-create-a-pull-request
 -->
+## 🍰 Pull request
+<!-- Describe the Pull request. What changes do you propose? -->
 
-<!-- WRITE A RELEVANT TITLE -->
 
-## Changes Proposed:
--  
--  
 
 
-## Issues Addressed:
-- Closes 
```

Comment:
Dont remove

Reply:
I did not remove it. It is replaced with

```
### Issues Addressed:
<!-- If the issue doesn't exist, describe it and how to reproduce it, please. If the issue already exists, just paste the link to the issue you close, like this: Closes https://github.com/azerothcore/azerothcore-wotlk/issues/967 -->
- None
```

And while `closes` is the keyword to autoclose the issue I think experienced developers tend to know that - besides there is alternatives like `fixes`. I analysed some PRs of yours and most just have a stray `Closes` in them while `None` is more precise in 95% of all cases.

Reply:
Personally i prefered to have Closes there. Anyone else got any thoughts?

Reply:
I agree with `None`. People should read the template and modify it accordingly. Some people don't do it and leave it "Closes" but it's their fault, not ours.